app/views/devise/shared/_links.html.haml

```diff
@@ -9,3 +9,7 @@
 - if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks'
   = link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name)
   %br
+
+- if devise_mapping.ichain_registerable?
+  = link_to "Sign up", new_ichain_registration_path(resource_name)
```

Comment:
Hm, ``new_ichain_registration_path(resource_name)`` should redirect you to the sign up page of the iChain provider, shouldn't it?